test/e2e/auth_test.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2021 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+package e2e
+
+import (
+	"testing"
+
+	"github.com/k14s/imgpkg/test/helpers"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBasicAuth(t *testing.T) {
```

Comment:
I would change these tests to also follow the `t.Run` layout we are using in other tests.

Reply:
done.